index.html

```diff
@@ -17,11 +18,24 @@
 <body>
 
  <!-- navigation code here -->
+  <nav class="nav">
```

Comment:
- seemes to be some nested media query styling that is shrinking the nav in the desktop view vs the tabled view.